src/main/java/ch/uzh/ifi/hase/soprafs22/rest/dto/chat/ChatOverViewGetDTO.java

```diff
@@ -3,10 +3,13 @@
 import ch.uzh.ifi.hase.soprafs22.entity.Message;
 import ch.uzh.ifi.hase.soprafs22.rest.dto.UserGetDTO;
 
+import java.util.Date;
+
 public class ChatOverViewGetDTO {
   private UserGetDTO user;
   private Message message;
   private Long chatId;
+    private Date matchCreationDate;
```

Comment:
Very nice. Very much wow.
![image](https://user-images.githubusercontent.com/75019468/167817581-ca0bd057-e3b1-4b51-9c42-51083322ff36.png)
No problemo in the client to get a Date object of the json string. TimeZone is handled perfectly.